stellar-contract-env-common/src/tuple.rs

```diff
@@ -39,7 +39,7 @@ macro_rules! impl_for_tuple {
         {
             fn into_env_val(self, env: &E) -> EnvVal<E, RawVal> {
                 let env = env.clone();
-                let vec = env.vec_new();
+                let vec = env.vec_new(RawVal::from_void());
```

Comment:
We should pass in `u32` `$count` here, as it is known. The syntax change I suggest is probably not quite right, because Rust will assume a number without a type will be an i32, but we need we actually need to build a u32 suffixed literal. The easy way to make it a u32 would be to add a u32 suffix to lines 50-61 to the first parameter.
```suggestion
                let vec = env.vec_new($count.into());
```